l1-contracts/contracts/state-transition/chain-deps/facets/Admin.sol

```diff
@@ -319,26 +311,14 @@ contract AdminFacet is ZkSyncHyperchainBase, IAdmin {
 
     // todo make internal. For now useful for testing
     function _prepareChainCommitment() public view returns (HyperchainCommitment memory commitment) {
+        require(s.priorityTree.unprocessedIndex > 0, "PQ not ready");
```

Comment:
I don't think it prevents from migrating when the PQ is not yet fully migrated to tree. When someone deposits a new transaction, the `s.priorityTree.unprocessedIndex` will grow beyond zero regardless of whether old txs were processed or not

Reply:
It won't. `s.priorityTree.unprocessedIndex` only grows when we execute batches from the tree.

`s.priorityTree.tree._nextLeafIndex` grows when someone deposits.